MYR_rails/app/assets/javascripts/application.js.erb

```diff
@@ -11,8 +11,7 @@
 // about supported directives.
 //
 //= require jquery
-//= require jquery.cookie
+//= require js.cookie
```

Comment:
I know this probably does not matter, but should this have changed?

Reply:
I think this is necessary with the change from the `jquery-cookie-rails` gem, which seemed to be incompatible with newer versions of rails, to the newer `js_cookie_rails` gem. But I don't have a good idea how all these bits fit together, so  let me know if I'm wrong about that!

(And thanks for looking at this)

Reply:
I'll take a look at the rest of the code to the best that I can in an hour or so 👍